array-manipulation-2/src/includes.js

```diff
@@ -0,0 +1,10 @@
+/* exported includes */
+function includes(array, value) {
+  var boolean = false;
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] === value) {
+      boolean = true;
```

Comment:
You can just `return` `true` here so you don't continue looping once the value has been found